analytics/Dockerfile

```diff
@@ -8,4 +8,4 @@ RUN pip install --no-cache-dir -r requirements.txt
 
 EXPOSE 6000
 
-CMD ["gunicorn", "-b", "0.0.0.0:5050", "app:app"]
+CMD ["gunicorn", "-b", "0.0.0.0:5050", "run:app"]
```

Comment:
sorry but what is the significance of changing this?

Reply:
It’s to ensure that the server looks in the right place to run the analytics service. I’ve changed the entry point to the app from `app.py` to `run.py`. 

So now it looks in `run.py` for `app` and that gets the analytics service up and running. 


More information about gunicorn is [here](https://flask.palletsprojects.com/en/3.0.x/deploying/gunicorn/)